requirements.txt

```
@@ -1,2 +1,2 @@
-prefect>=2.10.5
+prefect>=2.10.7
 docker>=6.0.0
```

A critical fix to docker-py was recently merged, but isn't in our Prefect images yet. Bumping this version should fix the test failures.
```suggestion
docker>=6.1.1
```

---

Thanks @desertaxle I'll try that.

---

That did the trick.